examples/bash/bash_in_docker/logic.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+set -x
+echo $(cat /dev/stdin)
```

Comment:
You don't even need the echo :)
Just `cat /dev/stdin` should work

Minimal example:
```
#!/bin/bash
set -x #optional logging
cat /dev/stdin
```

Reply:
yes, I know. I wanted to print something out on the client side.
Ok. I'm removing it.